api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -54,6 +54,32 @@ private Relationships() {
         owns
     }
 
+
+    /**
+     * The list of possible relationship (aka edges) direction. Relationships are not bidirectional.
+     */
+    public enum Direction {
+        /**
```

Comment:
Are the javadocs here correct? They seem to me a bit too concrete, because the meaning of a direction of a generic relationship is completely in the hands of the creator of the relationship.